src/options.cpp

```diff
@@ -456,7 +500,81 @@ bool cOpt::operator!=(const std::string sCompare) const
     return !(*this == sCompare);
 }
 
-void initOptions()
+/** Fill TILESETS mapping with values.
+ * Scans all directores in gfx directory for file named "tileset.txt".
```

Comment:
Actually it scans the `FILENAMES["gfxdir"]` directory for files names `FILENAMES["tileset-conf"]`. Both entries of the `FILENAMES` map may contain any string, not just "gfx" and "tileset.txt".